styles/new/base.css

```diff
@@ -0,0 +1,19 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+ul {
+  @apply list-disc font-open-sans font-regular text-neutral-900
+    dark:text-white;
+
+  font-style: normal;
```

Comment:
Please use Tailwind for these. There are Tailwind tokens for font-style and line-height (ie: https://tailwindcss.com/docs/line-height)